docs/userguides/accounts.md

```diff
@@ -0,0 +1,103 @@
+# Accounts
+
+Accounts in Ape come from [AccountAPI](../methoddocs/api.html#ape.api.accounts.AccountAPI) implementations (e.g. from plugins).
+There are typically two types of accounts:
+
+1. Test accounts
+2. Live network accounts
+
+Test accounts are useful for local network testing and debugging contracts.
+Live network accounts are for interacting with live blockchains and should be secured.
+
+To learn more about Ethereum accounts, see [the Ethereum documentation](https://ethereum.org/en/developers/docs/accounts/).
+
+## Test Accounts
+
+Ape ships with pytest fixtures to assist in writing your tests.
+Pre-funded test accounts are accessible via the [accounts fixture](./testing.html#accounts-fixture).
+
+```python
+def test_my_contract_method(accounts):
+    sender = accounts[0]
+    ...
+```
+
+To access the same prefunded accounts in your scripts or console, use the root `accounts` object and the [test_accounts](../methoddocs/managers.html#ape.managers.accounts.AccountManager.test_accounts) property:
+
+```python
+from ape import accounts
+
+sender = accounts.test_accounts[0]
+```
+
+You can configure your test accounts using your `ape-config.yaml` file:
+
+```yaml
+test:
+  mnemonic: test test test test test test test test test test test junk
+  number_of_accounts: 5
+```
+
+Learn more about test accounts from the [testing guide](./testing.html#accounts-fixture).
+
+## Live Network Accounts
+
+When using live networks, you need to get your accounts into Ape.
+Ape ships with a keyfile accounts plugin to assist with this.
+All the available CLI commands for the accounts plugin can be found [here](../commands/accounts.html).
+
+For example, you can [generate](../commands/accounts.html#accounts-generate) an account:
+
+```bash
+ape accounts generate <ALIAS>
+```
+
+It will prompt you for a passphrase.
+
+If you already have an account and you wish to import it into Ape (say, from Metamask), you can use the [import command](../commands/accounts.html#accounts-import):
+
+```bash
+ape accounts import <ALIAS>
+```
+
+It will prompt you for the private key.
+If you need help exporting your private key from Metamask, see [this guide](https://metamask.zendesk.com/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
+
+Then, in your scripts, you can [load](../methoddocs/managers.html#ape.managers.accounts.AccountManager.load) an account:
+
+```python
+from ape import accounts
+
+account = accounts.load("<ALIAS>")
+```
+
+## Automation
+
+If you use your keyfile accounts in automation, such as CI/CD, you may need to programmatically unlock them and enable autosign.
```

Comment:
I would actually change this to say "it is not recommended" to use them that way, because of the need to unlock, but it is possible

Reply:
Further, one day I hope the AWS KMS account plugin gets working, because that's actually a really good plugin type for CI/CD use cases, authorizing certain containers to have access to your managed signers in AWS (which is a better security workflow)

Reply:
I don't believe we have a ticket for this yet. I'll try to write something up in a ticket.

Reply:
I don't think it has really much priority to it, this is bundled more into the Silverback work (2023 goals)

Reply:
Resolved here [cb557fe](https://github.com/ApeWorX/ape/pull/981/commits/cb557fe2f55e7b2f1d686278688fb0fc9e28149d)